src/shapes/SurfaceRectangle.js

```diff
@@ -192,5 +192,16 @@ define([
 
         };
 
+        // Internal use only. Intentionally not documented.
+        SurfaceRectangle.prototype.getReferencePosition = function () {
+            return this.center;
+        };
+
+        // Internal use only. Intentionally not documented.
+        SurfaceRectangle.prototype.moveTo = function (globe, position) {
+            this.center = globe.computeShiftedLocations(this.getReferencePosition(), position,
```

Comment:
That's not what I had in mind. `this.center` is already a `Location` object. My questions was can we use it directly? Why are we creating a new `Location` object here? The same applies for `SurfaceEllipse` and `SurfaceCircle`.